gh-pages/doc/data-flow.html

```diff
@@ -0,0 +1,252 @@
+  <!DOCTYPE html>
```

Comment:
Why is this directory called gh-pages? Is this standard convention?
With this file converted to HTML, do we need the rappor/doc/ directory at all?